R/write.R

```diff
@@ -20,8 +20,8 @@
 #'   will append to existing file. In both cases, if file does not exist a new
 #'   file is created.
 #' @param col_names Write columns names at the top of the file?
-#' @param delim Delimiter used to separate values. Defaults to `" "`. Must be
-#'   a single character.
+#' @param delim Delimiter used to separate values. Defaults to `" "`. `write_excel_csv()` has `","` as default and
+#' `write_excel_csv2` has `";"` as default. Must be a single character.
 #' @param na String used for missing values. Defaults to NA. Missing values
 #'   will never be quoted; strings with the same value as `na` will
 #'   always be quoted.
```

Comment:
we should add an example for the new function

Reply:
but this would be e.g.: `write_excel_csv2(mtcars, 'some_path.csv')`. Do you thing is worth adding such example? write_excel_csv doesn't provide an example.